app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    inner_command = command.split()
```

Comment:
Neew to validate command
Your code will be broke if command equals to:
1. "mv"
2. "mv file1.txt"
3. ""